README_template.rst

```diff
@@ -25,5 +25,33 @@ Distributed registry for storing and querying health care providers their vendor
 Configuration
 *************
 
-.. include:: docs/pages/configuration/registry.rst
-    :start-after: .. marker-for-readme
+Sync modes
```

Comment:
Shouldn't this be in the docs as well and thus be an include?

Reply:
You're right, fixed!